getstarted.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default-nav
+title: Get Started
+description: mlpack | fast, flexible machine learning library in C++
+order: 2
+---
+
+<div class="page-title-header">Get Started</div>
+
+<h1>Overview</h1>
+
+There are multiple ways to get mlpack up and running. Python bindings can be installed using pip or conda, or built from source (see "Build from source" tutorials).
+For C++, if mlpack is not available via your preferred OS package manager, or if you need to build your own version (e.g. to apply optimizations, use a different 
+set of BLAS/LAPACK, or build a different configuration), please refer to the "Build from source" tutorials.
+For Windows, prebuilt binaries will help you get started without the need of building mlpack. These packages include both the C++ mlpack library as well as the CLI tools.
+
+Here is a summary of the currently available distribution options you can use depending on your needs:
+
+<h2>Python</h2>
```

Comment:
In this file also I think we can replace all the `<hX>` with Markdown headings, and the links with Markdown links.

Reply:
Done. Could you please review all the links and update them accordingly? When I started this project we were referring to mlpack 3.1 and new versions have been released. Also, I believe we should be using a static link instead of versioned (i.e. stable).

Reply:
I didn't do my homework, which was to figure out how to push the current state of mlpack.org back to the repository.  I'll do that now, but I know it will cause merge conflicts on this branch.  I can work them out too and push the changes / provide a patch, or just work it out during the actual merge of the PR; up to you.

Reply:
I think it will be faster if you solve the conflicts and/or manually update the files to use the correct links. Then moving forward we can make sure it's a seamless process.